builder/builder-git.c

```diff
@@ -56,6 +56,19 @@ git_get_mirror_dir (const char     *url_or_path,
   g_autoptr(GFile) git_dir = NULL;
   g_autofree char *filename = NULL;
   g_autofree char *git_dir_path = NULL;
+  GPtrArray *sources_dirs = NULL;
+  int i;
+
+  sources_dirs = builder_context_get_sources_dirs (context);
+  for (i = 0; i < sources_dirs->len; i++)
```

Comment:
Ok, it was not for me since I was always a pointer array. Fixed that and added a clause for the loop accordingly for git/file/archive.